src/control/lib/ui/flags.go

```diff
@@ -78,9 +83,13 @@ func (f *LabelOrUUIDFlag) UnmarshalFlag(fv string) error {
 }
 
 const (
-	// FIXME: Pin these to something in the engine code
-	maxKeyLen   = 20
-	maxValueLen = 128
+	// MaxPropKeyLen is the maximum length of a property key.
+	MaxPropKeyLen = 20
+	// MaxPropValLen is the maximum length of a property value.
+	MaxPropValLen = 128
+
+	propKvSep = ":"
+	propSep   = ","
```

Comment:
Nit - still using the word prop here when most of the code has been changed to say "attrs"

Reply:
Well... The flag types in this package are named `(Get|Set)PropertiesFlag`. They were originally developed to provide base functionality for the pool/container properties managed via the `dmg` and `daos` tools. It's a little fuzzy, I guess one could argue that a Property is a special class of Attribute with defined keys and values. One could also argue that an Attribute is a special class of Property that allows arbitrary keys and values. ;) I'm inclined to go with the latter because it requires less renaming.

Reply:
I can go either way, my hope is to avoid being too confusing for folks familiar with the DAOS prop/attr concepts.